src/h5web/visualizations/index.tsx

```diff
@@ -86,8 +84,8 @@ export const VIS_DEFS: Record<Vis, VisDef> = {
     Container: LineVisContainer,
     supportsEntity: (entity) => {
       return (
-        isMyDataset(entity) &&
-        hasMyNumericType(entity) &&
+        isDataset(entity) &&
+        hasNumericType(entity) &&
         hasMySimpleShape(entity) &&
```

Comment:
It's only used exactly like this twice -- the Matrix vis calls `hasBaseType` instead of `hasNumericType`, so not sure it's worth it. Unless you feel strongly about it? Wouldn't know what to call it anyway 😂

Reply:
Ah yes, these are assertions in the containers... Nevermind